src/schnetpack/nn/nuclear_embedding.py

```diff
@@ -0,0 +1,87 @@
+import math
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from schnetpack.electron_configurations import electron_config
+
+
+class NuclearEmbedding(nn.Module):
+    """
+    Embedding which maps scalar nuclear charges Z to vectors in a
+    (num_features)-dimensional feature space. The embedding consists of a freely
+    learnable parameter matrix [Zmax, num_features] and a learned linear mapping
+    from the electron configuration to a (num_features)-dimensional vector. The
+    latter part encourages alchemically meaningful representations without
+    restricting the expressivity of learned embeddings.
+    Using complexe nuclear embedding can have negative impact on the model performance, when spin charge embedding is activated
+
+    Arguments:
```

Comment:
again arguments in docstring do not correspond to __init__ method arguments

Reply:
done in 800c3b0